NetKet/Graph/abstract_graph.hpp

```diff
@@ -52,9 +52,10 @@ class AbstractGraph {
   virtual int Size() const noexcept = 0;
 
   /**
-  Returns the graph edges.
+  Member function returning the graph edges.
+  @return Graph edges.
   */
-  virtual std::vector<Edge> const &Edges() const noexcept = 0;
+  virtual std::vector<Edge> const &Edges() const = 0;
```

Comment:
Why is that? Could you revert it, please?

Reply:
I've cut too many noexcept...thanks